openedx/core/djangoapps/course_date_signals/tests.py

```diff
@@ -196,6 +196,15 @@ def test_get_custom_pacing_children(self):
             self.assertCountEqual(_get_custom_pacing_children(sequence, 2), expected_dates)
 
 
+def override_waffle_flag_class(decorator):
```

Comment:
This decorator doesn't have to be just used for overriding a waffle flag, but can be used for other decorators as well. So this function name can be misleading.

Reply:
And I think this function would ideally [go in this file](https://github.com/edx/edx-toggles/blob/5d59cc4e9429a96179caefd3af9cc9d84e6d1cef/edx_toggles/toggles/testutils.py)